modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -66,3 +68,37 @@ def monotonic():
             so that only the difference between the results of consecutive calls is valid.
         """
         return monotonic()
+
+    def copy_ignite_work_dir(self):
```

Comment:
too strange place to works with Ignite internals

Reply:
why strange? IgniteTest.
i think igniteService doesn't need to know TestContext